kitsune/sumo/static/sumo/js/libs/uitour.js

```diff
@@ -2,272 +2,828 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Any incompatibilities with what we currently have?

Reply:
The only other piece of UITour we're using was `resetFirefox` and that still works fine!